assets/src/scss/layout/_footer.scss

```diff
@@ -13,11 +13,19 @@
   line-height: 0;
 
   a _-- {
-    color: inherit;
+    color: $white;
+
+    &:visited {
```

Comment:
You could combine the selectors into one, to simplify and shorten the code a bit? Something like:
```
&:visited,
&:hover,
&:active {
   color: $white;
}
```